client/src/Components/Pages/Questionnaire/BasicDetails1/index.js

```diff
@@ -1,74 +1,75 @@
-      scoolEmail: '',
-      phoneNum: '',
+    constructor(props){
+        super(props)
+        this.state = {
+            grade: '',
+            city: '',
+            scoolEmail: '',
+            phoneNum: '',
+
+        }
+    }
 
-    };
-  }
+  onChange = (e) => {
+    const { name, value } = e.target;
+    this.setState({ 
+        [name]: value
+    });
+  };
+    
+    render() {
 
-  render() {
-    const storage = localStorage.getItem('state') || '{}';
-    const parsedStorage = JSON.parse(storage);
-    const {
-      grade, city, schoolEmail, phoneNum,
-    } = parsedStorage;
-    return (
-      <form onSubmit={this.handleSubmit}>
-        <div className="basic-details1">
-          <h2>Hello, Please fill out these basic details and then we will move on to the interesting stuff</h2>
-          <div className="basic-details1-content">
-            <div className="basic1-content1">
-              <p>What is your current Year Group at UK School?</p>
-              <input
-                type="text"
-                className="sec1-input"
-                name="grade"
-                onChange={this.props.handleChange}
-                placeholder="Enter your grade .."
-              />
-              <p>
-In which city exactly do you live?
-                <span className="special-txt">(we need this to work out how far you have to travel to your workexperience if needed)</span>
-              </p>
-              <input
-                type="text"
-                className="sec1-input"
-                name="city"
-                onChange={this.props.handleChange}
-                placeholder="Enter your city name .."
-              />
-            </div>
-            <div className="basic1-content2">
-              <p>School email address</p>
-              <input
-                type="email"
-                className="sec1-input"
-                name="schoolEmail"
-                onChange={this.props.handleChange}
-                placeholder="***@**.edu"
-              />
-              <p>Enter your phone number</p>
-              <input
-                type="tel"
-                className="sec1-input"
-                name="phoneNum"
-                onChange={this.props.handleChange}
-                placeholder="Enter your phone number"
-              />
-            </div>
-          </div>
-        </div>
-      </form>
-    );
-  }
+        return (
+                <form onSubmit={this.handleSubmit}>
+                    <div className='basic-details1'>
+                        <h2>Hello, Please fill out these basic details and then we will move on to the interesting stuff</h2>
+                            <div className="basic-details1-content">
+                                <div className="basic1-content1">
+                                    <p>What is your current Year Group at UK School?</p>
+                                    <input 
+                                        type="text" 
```

Comment:
it's more usability if we add a select list
year  7
year 8
year 9
year 10
year 11